diff --git a/research/object_detection/core/box_list_ops.py b/research/object_detection/core/box_list_ops.py
index cb457b728..cd7adfc2f 100644
--- a/research/object_detection/core/box_list_ops.py
+++ b/research/object_detection/core/box_list_ops.py
@@ -842,7 +842,7 @@ def _copy_extra_fields(boxlist_to_copy_to, boxlist_to_copy_from):
 
 
 def to_normalized_coordinates(boxlist, height, width,
-                              check_range=True, scope=None):
+                              check_range=False, scope=None):
   """Converts absolute box coordinates to normalized coordinates in [0, 1].
 
   Usually one uses the dynamic shape of the image or conv-layer tensor:
@@ -881,7 +881,7 @@ def to_normalized_coordinates(boxlist, height, width,
 def to_absolute_coordinates(boxlist,
                             height,
                             width,
-                            check_range=True,
+                            check_range=False,
                             maximum_normalized_coordinate=1.1,
                             scope=None):
   """Converts normalized box coordinates to absolute pixel coordinates.
